### PR TITLE
more docs on env vars

### LINKED ISSUE #891: Improve env var docs
Maybe in the configuration section of the docs, go through each env var we're using and say how.

### PR DESCRIPTION
**Pull request by sastels**
resolves #891

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,49 +8,37 @@ https://cdsvacpoc.azurewebsites.net/
...
+## Environment variables
+
+These need to be set on the production service for the app to function correctly.
+Some are also required for testing. You will also need some of these set for local development
+(at the very least, you should have `AIRTABLE_READ_KEY` and `GOOGLE_MAPS_KEY`).
+Contact other developers on the project for what values we're currently using.
+
+| Variable                     | Use                                                     | Where                           |
+| ---------------------------- | ------------------------------------------------------- | ------------------------------- |
+| `AIRTABLE_READ_KEY`          | load data (benefits / translations / etc) from Airtable | production / locally            |
```
Comment (maxneuvians) [inline README.md:20]:
```
import random

keys = [AIRTABLE_READ_KEY,AIRTABLE_WRITE_KEY,BROWSERSTACK_ACCESS_KEY,BROWSERSTACK_USERNAME,GA_UA,GITHUB_PUBLIC_ACCESS_TOKEN,GOOGLE_MAPS_KEY,SENTRY_DSN]

print random.shuffle(keys)
```

**Review by maxneuvians — APPROVED** [review]
🦑 LGTM - one small suggestion 👍

File: README.md (hunk 1 above)
Comment (sastels) [reply README.md:20]: I R confuzd 🙃

Comment (maxneuvians) [reply README.md:20]: You changed the key order from alphabetical to random 😈

Comment (sastels) [reply README.md:20]: not random - by importance :)

**Pull request merged**